pom.xml

```diff
@@ -371,6 +373,90 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>license-maven-plugin</artifactId>
+                <version>2.0.0</version>
+                <executions>
+                    <execution>
+                        <id>licenses-report</id>
+                        <phase>prepare-package</phase>
+                        <goals>
+                            <goal>aggregate-download-licenses</goal>
+                            <goal>add-third-party</goal>
+                            <goal>aggregate-add-third-party</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <fileTemplate>${project.basedir}/code-style/NOTICES.ftl</fileTemplate>
+                    <outputDirectory>${project.basedir}</outputDirectory>
+                    <thirdPartyFilename>NOTICES.md</thirdPartyFilename>
+                    <licensesOutputDirectory>${project.basedir}/licenses</licensesOutputDirectory>
+                    <includeTransitiveDependencies>false</includeTransitiveDependencies>
+                    <cleanLicensesOutputDirectory>true</cleanLicensesOutputDirectory>
+                    <!--
+                     Normalise equivalent license names (see also NOTICES.ftl)
+                     Format is "<preferred license name> :: <file name>|<license name>|<license name>|..."
+                    -->
+                    <licenseMerges>
+                        <licenseMerge>Apache Software License 2.0 :: apache_software_license_2.0.txt|Apache Software License, Version 2.0|The Apache Software License, Version 2.0|Apache License, Version 2.0|Apache License 2.0|Apache 2|Apache 2.0</licenseMerge>
+                        <licenseMerge>MIT License :: mit_license.html|MIT License|The MIT License|MIT X11 License</licenseMerge>
+                        <licenseMerge>Eclipse Public License 1.0 :: eclipse_public_license_1.0.html|Eclipse Public License 1.0|The Eclipse Public License 1.0|Eclipse Public License - v 1.0|Eclipse Distribution License - v 1.0</licenseMerge>
+                        <licenseMerge>BSD License 3 :: bsd_license_3.html|BSD Licence 3</licenseMerge>
+                        <licenseMerge>GNU Lesser General Public License 2.1 :: gnu_lgpl_2.1.html|GNU Lesser General Public License</licenseMerge>
+                        <licenseMerge>Mozilla Public License 2.0 or Eclipse Public License 1.0 :: mpl_2.0_or_epl_1.0.html|MPL 2.0 or EPL 1.0</licenseMerge>
+                        <licenseMerge>Common Development and Distribution License (CDDL) 1.1 :: cddl_gpl_1.1.txt|CDDL 1.1</licenseMerge>
+                        <licenseMerge>CDDL and GPL License :: cddl+gpl_1.1.html|CDDL+GPL License</licenseMerge>v
```

Comment:
Same license

Reply:
Seems fine in services, just forgot to backport that change?

Reply:
Yep, this has now been backported from services.